Fedora/input/xccdf/system/accounts/physical.xml

```diff
@@ -176,7 +203,8 @@ the case of mixed OS environment, this can create the risk of short-term
 loss of availability of systems due to unintentional reboot.
 </rationale>
 <oval id="disable_ctrlaltdel_reboot" />
-<!--ref nist="IA-2(1),AC-3" disa="213" /-->
+<ident cce="27511-5" />
```

Comment:
The CCE-id should be removed from Fedora content (since I assume it's RHEL-7 specific).